giotto/base.py

```diff
@@ -1,6 +1,6 @@
 """Implements a TransformerResamplerMixin for transformers that have a resample
```

Comment:
Are these comments really needed? Ideally the code itself would guide the reader and removing these comments would save on needing to maintain them (e.g. if the module is extended).

Reply:
It's required by flake8, this I recall is the main reason we have these docstrings.